index.js

```diff
@@ -10,34 +10,28 @@ app.set("view engine", "ejs") // set ejs as view engine
 app.set("views", path.join(__dirname, "views"))
 app.use(express.static(path.join(__dirname, "public")))
 
-// const urls = [
-//   "https://api.cryptonator.com/api/ticker/btc-usd",
-//   "https://api.cryptonator.com/api/ticker/eth-usd",
-//   "https://api.cryptonator.com/api/ticker/ada-usd",
-// ];
-
-const btcUrl = "https://api.cryptonator.com/api/ticker/btc-usd"
-const ethUrl = "https://api.cryptonator.com/api/ticker/eth-usd"
-const adaUrl = "https://api.cryptonator.com/api/ticker/ada-usd"
+const urls = [
+  "https://api.cryptonator.com/api/ticker/btc-usd",
+  "https://api.cryptonator.com/api/ticker/eth-usd",
+  "https://api.cryptonator.com/api/ticker/ada-usd",
+]
 
 var btcUsdObj
 var ethUsdObj
 var adaUsdObj
 
-fetch(btcUrl)
-  .then((response) => response.json())
-  .then((data) => (btcUsdObj = data.ticker))
-  .then(() => console.log(btcUsdObj)) // for terminal feedback
-
-fetch(ethUrl)
-  .then((response) => response.json())
-  .then((data) => (ethUsdObj = data.ticker))
-  .then(() => console.log(ethUsdObj)) // for terminal feedback
-
-fetch(adaUrl)
-  .then((response) => response.json())
-  .then((data) => (adaUsdObj = data.ticker))
-  .then(() => console.log(adaUsdObj)) // for terminal feedback
+Promise.all(
+  urls.map((url) => {
+    return fetch(url).then((resp) => resp.json())
+  })
```

Comment:
Great use of `Promise.all`. 

You could package the whole `Promise.all` logic in an `async` function. The function could return an object like this:

```js
{
  btcUsd: {/* result */},
  ethUsd: {/* result */},
  adaUsd: {/* result */},
}
```